data/endpoints/chaps-bank-to-bank-v4.json

```diff
@@ -278,11 +278,13 @@
                     },
                     "currency": {
                         "minLength": 1,
-                        "type": "string"
+                        "type": "string",
+                        "description":"Set to 'GBP' as other currencies are not supported.",
+                        "example": "GBP"
                     }
                 },
                 "additionalProperties": false,
-                "description": "A number of monetary units specified in an active or a historic currency where the unit of currency is explicit and compliant with ISO 4217."
+                "description": "Amount of money to be moved between the debtor and creditor, before deduction of charges, expressed in the currency as ordered by the initiating party."
```

Comment:
Not sure if the 'before deduction of changes' is necessary as charges are deducted currently, but may be needed for future or as part of a payment chain?

Reply:
Confirmed with product owner that this is the desired wording.